Fix a comment

lesv opened the pull request:
1. Remove mention of Jackson - since there isn’t an endpoint in this
   version.
2. Upgrade to latest version
3. Remove non-germane code.

Commit:
Fix a comment

1. Remove mention of Jackson - since there isn’t an endpoint in this
version.
2. Upgrade to latest version
3. Remove non-germane code.

lesv commented:
4. add START / END tags

lesv commented:
@oalami

oalami commented on final/src/main/java/com/example/guestbook/Greeting.java:
```diff
@@ -35,8 +36,7 @@
  *
- * NOTE - all the properties are PUBLIC so that can keep this simple, otherwise,
- * Jackson, wants us to write a BeanSerializaer for cloud endpoints.
+ * NOTE - all the properties are PUBLIC so that can keep the code simple.
```
grammar (all the properties are public to keep the code simpler). 

Also, please clean up this comment.
1. Mention of OfyHelper should be a javadoc link
1. Clean up the sentence structure, ex: Must be need to register it with OfyHelper. (I don't like the -- very important)
1. "huge win in performance" - likely more considerations on when to index or not, this comment could lead to someone choosing to index everything.

oalami commented on final/src/main/java/com/example/guestbook/GuestbookServlet.java:
```diff
@@ -31,22 +32,14 @@
   @Override
   public void doGet(HttpServletRequest req, HttpServletResponse resp)
```
add comment

oalami commented:
@lesv minor comments otherwise lgtm :+1:

Commit:
Fix comments.

1. Improve the language
2. Get rid of pointless servlet
3. Add comments to SignGuestbookServlet
